content/en/llm_observability/configuration/_index.md

```diff
@@ -21,28 +21,55 @@ Topics
+## Connect your OpenAI account
 
-{{< img src="llm_observability/configuration.png" alt="An example of an LLM application's configuration settings in LLM Observability" style="width:100%;" >}}
+Connect your OpenAI account to LLM Observability with your OpenAI API key.
 
-Enabling any of the [out-of-the-box evaluations](#select-evaluations) outside of `Language Mismatch` shares your input and output to OpenAI. 
+1. In Datadog, navigate to [**LLM Observability > Settings > Integrations**][2]. 
+1. Select **Connect** on the OpenAI tile.
+1. Follow the instructions on the tile. 
+   - Provide your OpenAI API key. Ensure that this key has **write** permission for **model capabilities**.
+1. Enable **Invoke Model from this API key**.
 
-## Enter a topic
+{{< img src="llm_observability/configuration/openai-tile.png" alt="The OpenAI configuration tile in LLM Observability. Lists instructions for configuring OpenAI and providing your OpenAI API key." style="width:100%;" >}}
 
-To enter a topic, click the Edit icon and add keywords. For example, for an LLM application that was designed for incident management, add `observability`, `software engineering`, or `incident resolution`. 
+## Select and enable evaluations
 
-Topics can contain multiple words and should be as specific and descriptive as possible. For example, if your application handles customer inquiries for an e-commerce store, you can use "Customer questions about purchasing furniture on an e-commerce store".
+Navigate to [**LLM Observability > Settings > Evaluations**][3]. 
 
-## Select evaluations
+{{< img src="llm_observability/configuration/settings.png" alt="The Evaluations tab, featuring a list of existing evaluations." style="width:100%;" >}}
 
-To enable evaluations, click the toggle for the respective evaluations that you'd like to assess your LLM application against in the Quality and Security and Safety sections. For more information about evaluations, see [Terms and Concepts][1].
+1. Click on the evaluation you want to enable.
+1. Select **OpenAI** as your LLM provider.
+1. Select the OpenAI account you want to run the evaluation on.
+1. Assign the LLM application you want to run the evaluation on.
+
+After you click **Save**, LLM Observability invokes a `GPT-4o mini` model using the OpenAI API key you provided.
+
+You can monitor the usage of this API key by querying for the metrics `ml_obs.span.llm.input.tokens`, `ml_obs.span.llm.output.tokens`, and `ml_obs.span.llm.total.tokens`. Filter by the `evaluation:default` tag.
+
+For more information about evaluations, see [Terms and Concepts][1].
+
+## Provide topics for topic relevancy
+
+Providing topics allows you to use the [topic relevancy][4] evaluation. 
+
+1. Go to [**LLM Observability > Applications**][5].
+1. Select the application you want to add topics for.
+1. At the bottom of the left sidebar, select **Configuration**.
+1. Add topics in the pop-up modal.
```

Comment:
Hi @barieom, can you confirm that this is the correct location to enter topics?

Reply:
Appears to be correct in app.datadoghq!

Reply:
yes it's correct!